src/core/Toast/Toast.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { baseStyles } from './Toast.baseStyles';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+import { HtmlDiv, HtmlDivWithRef } from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+
+export interface ToastProps {
+  /** Set aria-live mode for the Toast text content and label.
+   * @default 'polite'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Label heading for the Toast */
+  headingText?: string;
+  /** Text content of the Toast. */
+  children?: ReactNode;
+  /** Custom classname to extend or customize */
```

Comment:
This description is not that clear in my opinion even though I think it is used elsewhere as well. I think "Custom class name for styling and customizing" or something along those lines would be clearer.

Reply:
Fixed, changed JS docs comments